docs/ignoring-files-folders-code.md

```diff
@@ -184,10 +183,17 @@ bad_func3(   // nosemgrep: configs.rule-id-3, configs.rule-id-4
 Previous annotations for ignoring code inline, such as `nosem`, are deprecated.
 :::
 
-## Disabling rules on Semgrep Cloud Platform
+## Disable rules on Semgrep Cloud Platform
 
 Semgrep Cloud Platform users can disable rules and rulesets through the Policies page. See [Disabling rules](/semgrep-code/policies#disabling-rules) and [Disabling rulesets](/semgrep-code/triage-remediation/#disable-a-ruleset-or-a-rule).
 
+## Ignore findings
+
+See [Triage and remediate Semgrep Code findings in Semgrep Cloud Platform](/semgrep-code/triage-remediation/#ignore-findings) to learn how to:
```

Comment:
```suggestion
You can also ignore findings as a triage action. In this case, a finding record is generated rather than excluded. See [Triage and remediate Semgrep Code findings in Semgrep Cloud Platform](/semgrep-code/triage-remediation/#ignore-findings) to learn how to:
```